content/docs/1.6.0/deploy/install/install-with-fleet.md

```diff
@@ -0,0 +1,153 @@
+
+> Use [this script](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/scripts/environment_check.sh) to check the Longhorn environment for potential issues.
+
+## Installing Longhorn
+
+1. In your GitOps repository, create a [fleet.yaml](https://fleet.rancher.io/ref-fleet-yaml) file that includes the following:
+
+    - Parameter for installing Longhorn in the `longhorn-system` namespace
+
+    ```yaml
+    defaultNamespace: longhorn-system
+    ```
+
+    - Parameters for [ignoring modified CRDs](https://fleet.rancher.io/bundle-diffs)
+
+    ```yaml
+    diff:
+      comparePatches:
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: engineimages.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: nodes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: volumes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+    ```
+
+    - Parameters for specifying the version of the Longhorn Helm chart to be installed
+
+    ```yaml
+    helm:
+      repo: https://charts.longhorn.io
+      chart: longhorn
+      version: v1.6.0 # Replace with the Longhorn version you'd like to install or upgrade to
+      releaseName: longhorn
+    ```
+
+    Example of a complete `fleet.yaml` file:
+
+    ```yaml
+    defaultNamespace: longhorn-system
+    helm:
+      repo: https://charts.longhorn.io
+      chart: longhorn
+      version: 1.6.0
+      releaseName: longhorn
+    diff:
+      comparePatches:
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: engineimages.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: nodes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+      - apiVersion: apiextensions.k8s.io/v1
+        kind: CustomResourceDefinition
+        name: volumes.longhorn.io
+        operations:
+        - {"op": "replace", "path": "/status"}
+    ```
+
+1. Create a GitRepo custom resource (CR) that points to your GitOps repository.
+
+    ```bash
+    cat > longhorn-gitrepo.yaml << "EOF"
+    apiVersion: fleet.cattle.io/v1alpha1
+    kind: GitRepo
+    metadata:
+      name: longhorn
+      namespace: fleet-local
+    spec:
+      repo: https://github.com/your-username/your-gitops-repo.git
+      revision: main
+      paths:
+      - .
+    EOF
+    ```
+
+1. Apply the GitRepo CR.
+
+    ```bash
+    kubectl apply -f longhorn-gitrepo.yaml
+    ```
+
+1. Verify that the GitRepo CR was created and synced successfully.
+
+    ```bash
+    kubectl -n fleet-local get fleet -w
```

Comment:
You didn't mention creating the namespace `fleet-local`. I would suggest just using the `default` namespace, good enough for an example.

Reply:
This follows [Fleet document](https://fleet.rancher.io/quickstart) and could be what it expects. Should we use `default` instead?

Reply:
Then, let's just need to add creating the ns beforehand.

Reply:
Namespace `fleet-local` is automatically created during `fleet` installation when `helm install fleet` is executed.

Reply:
Then, we are good